handlers/main.yml

```diff
@@ -51,3 +60,15 @@
 - name: reload dconf
   become: yes
   command: dconf update
+
+- name: restart auditd
+  shell: /sbin/service auditd restart
```

Comment:
We should use the `command` module here. You should prefer the `command` module over `shell` unless you absolutely need Ansible to run the command in a shell. It is rare that you need that though.

Reply:
Fixed now